mmcv/runner/hooks/evaluation.py

```diff
@@ -378,6 +404,12 @@ class DistEvalHook(EvalHook):
         broadcast_bn_buffer (bool): Whether to broadcast the
             buffer(running_mean and running_var) of rank 0 to other rank
             before evaluation. Default: True.
+        out_dir (str, optional): The root directory to save checkpoints. If not
+            specified, `runner.work_dir` will be used by default. If specified,
```

Comment:
'`runner.work_dir`' -> '``runner.work_dir``' to highlight it as a variable in doc.

Reply:
got it